BesterLive/AppDelegate.swift

```diff
@@ -14,7 +14,7 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
   var window: UIWindow?
 
 
-  func application(application: UIApplication, didFinishLaunchingWithOptions launchOptions: [NSObject: AnyObject]?) ->Bool{
+  func application(application:UIApplication, didFinishLaunchingWithOptions launchOptions: [NSObject:AnyObject]?) ->Bool{
```

Comment:
Reporter: CHECKSTYLE
Rule: error
Severity: WARN
File: /Users/framgia/.jenkins/jobs/BesterLive/workspace/BesterLive/AppDelegate.swift L17
 

Colons should be next to the identifier when specifying a type.

<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>